sources/reference/tutorials/batch.md

```diff
@@ -287,73 +412,192 @@ By creating a job role for our Job, we can:
-* Select `Additional configuration`
-* Select `ctk-batch-job-role` from the `Job role` dropdown
-* Scroll down to the `Log configuration` section
-* Select `awslogs` from the `Log driver` dropdown
-* Scroll down to the `Tags` section
-* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
-* Select `Enable` under `Propagate Tags`
-* Select `Create`
-
-[this StackOverflow post]: https://stackoverflow.com/a/66802973
+=== "AWS Console"
+
+    * Navigate to the Batch console and select `Job definitions`
+    * Select `Create`
+    * Provide a name
+        * For the purpose of this guide we'll use `ctk-batch-job-def`
+    * Leave `Fargate` selected
+    * Scroll down to `Container properties`
+    * For `Image`, provide the URI of your ECR image from earlier
+        * For example, when writing this guide, ours is:
+    `<our-aws-account-id>.dkr.ecr.eu-west-2.amazonaws.com/ctk-batch:latest`
+    * Leave `Bash` selected and inside command put: `run experiment-1.json`
+    * Select `0.25` for `vCpus` and `0.5 GB` for `Memory`
+    * Select `ctk-batch-execution-role` from the `Execution role` dropdown
+    * If your Compute environment is using **Public** subnets, select `Assign public
+    IP`
+        * If you're using **Private** subnets, please follow [this StackOverflow post][]
+        on what best setup to have to enable your jobs to communicate with ECR
+    * Select `Additional configuration`
+    * Select `ctk-batch-job-role` from the `Job role` dropdown
+    * Scroll down to the `Log configuration` section
+    * Select `awslogs` from the `Log driver` dropdown
+    * Scroll down to the `Tags` section
+    * Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+    * Select `Enable` under `Propagate Tags`
+    * Select `Create`
+
+    [this StackOverflow post]: https://stackoverflow.com/a/66802973
+
+=== "AWS CLI"
+
+    * Create a file named `container-properties.json` with the following contents:
+    ```json
+    {
+        "image": "ECR_IMAGE_URI",
+        "command": ["run", "experiment-1.json"],
+        "jobRoleArn": "JOB_ROLE_ARN",
+        "executionRoleArn": "EXECUTION_ROLE_ARN",
+        "resourceRequirements": [
+            {
+                "value": "512",
+                "type": "MEMORY"
+            },
+            {
+                "value": "0.25",
+                "type": "VCPU"
+            }
+        ],
+        "logConfiguration": {
+            "logDriver": "awslogs"
+        },
+        "networkConfiguration": {
+            "assignPublicIp": "ENABLED"
+        },
+        "fargatePlatformConfiguration": {
+            "platformVersion": "LATEST"
+        }
+    }
+    ```
+
+    * Replace `ECR_IMAGE_URI` in the file with the URI of the image you pushed
+    to ECR. Replace `JOB_ROLE_ARN` with the ARN of `ctk-batch-job-role`. Replace
+    `EXECUTION_ROLE_ARN` with the ARN of `ctk-batch-execution-role`.
```

Comment:
Similarly, can we replace `EXECUTION_ROLE_ARN` with `arn:aws:iam::<your-aws-account-id>:role/ctk-batch-execution-role`

Reply:
Yep, done